src/sagemaker_containers/env.py

```diff
@@ -436,7 +438,9 @@ def __init__(self):
         resource_config = read_resource_config()
         current_host = resource_config['current_host']
         hosts = resource_config['hosts']
+        network_interface_name = resource_config.get('network_interface_name', 'ethwe')
```

Comment:
is it safe to have a (non-None) default for this?

Reply:
This isn't actually added to resourceconfig yet, but the network interface name is needed for distributed training. As I see it, the choice is having a default here vs. having a default downstream and changing the value once this value changes.